src/supertux/command_line_arguments.cpp

```diff
@@ -115,6 +116,7 @@ CommandLineArguments::print_help(const char* arg0) const
     << _("Game Options:") << "\n"
     << _("  --edit-level                 Open given level in editor") << "\n"
     << _("  --resave                     Loads given level and saves it") << "\n"
+    << _("  --from-datadir               Indicates that the LEVELFILE path is relative to the data dir. Only used if --edit-level is specified") << "\n"
```

Comment:
IMO just `--from-datadir` isn't clear enough, it leaves confusion as to whether only the following path is from the datadir or if all of them are. To fix this, I'd suggest `--paths-from-datadir` (It also makes it clear in an instant that this is about paths).

I also recommend changing "LEVELFILE path" to just "paths" <sub>(plural)</sub> since all paths on the CLI should be affected by this.